test/CMakeLists.txt

```diff
@@ -559,7 +559,7 @@ if(CELERITAS_CORE_GEO STREQUAL "Geant4")
   set(_fails_g4geo DISABLE)
 endif()
 
-celeritas_add_test(celeritas/user/DetectorSteps.test.cc GPU)
+celeritas_add_test(celeritas/user/DetectorSteps.test.cc LINK_LIBRARIES CUDA::cudart GPU)
```

Comment:
Not sure what's the best way to handle the dependency to cudart since the allocator uses cuda to alloc host memory